packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -496,6 +496,11 @@ msgstr ""
 msgid "Back"
 msgstr "Enrere"
 
+#. Default: "Bad request"
```

Comment:
It would be more helpful to tell the user what do after getting this error message than just "Bad request." Should they contact their website administrator? Make a sacrifice to the gods? Cry out in anguish? All the above?

```suggestion
#. Default: "Bad request."
```